src/metrics/family.rs

```diff
@@ -119,7 +129,7 @@ impl<S: Clone + std::hash::Hash + Eq, M: Default> Default for Family<S, M> {
     }
 }
 
-impl<S: Clone + std::hash::Hash + Eq, M> Family<S, M> {
+impl<S: Clone + std::hash::Hash + Eq, M, C: MetricConstructor<M>> Family<S, M, C> {
```

Comment:
```suggestion
impl<S: Clone + std::hash::Hash + Eq, M, C> Family<S, M, C> {
```

Is this bound needed?